### PR TITLE
Feature/new tracks screen design

### PR DESCRIPTION
**Pull request by DominikGuzei**
This PR implements the new tracks screen design. Here is a short summary of the changes:
- New graphics / buttons / font etc.
- There are only 3 tracks per page now
- To display the username of the track owner the tracks serializer includes a custom attribute "username" that retrieves the name of the user associated to the track
- New button "Create a Track" on the tracks screen

### CODE REVIEW

**Comment by stravid** [comment]
Tag

What do we do with long track names? Either we make them brake over several lines or we just truncate them. I'm for the first option :-)

**Comment by stravid** [comment]
Great! Mergeable in my opinion!

File: app/assets/javascripts/slotcars/tracks/templates/page_view_template.js.hjs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {{#each tracks}}
   <div class="track" data-track-id="{{unbound id}}">
-    <h2 class="track-title">{{title}}</h2>
     {{view Shared.ThumbnailTrackView trackBinding="this" class="track-container" }}
+    <h3>{{title}}</h2>
+    <h4>{{username}}
```
Comment (roboshoes) [inline app/assets/javascripts/slotcars/tracks/templates/page_view_template.js.hjs]: is it just me or do you not close the `<h4>` tag?

Comment (roboshoes) [inline app/assets/javascripts/slotcars/tracks/templates/page_view_template.js.hjs]: also ... you open a `<h3>` and close a `</h2>`

File: app/assets/stylesheets/views/tracks_screen_view.css.scss [hunk 2]
```diff
@@ -1,30 +1,127 @@
...
+  }
+
+  h4 {
+    width: 80%;
+    text-align: center;
+    font-family: "JustTheWayYouAre", sans-serif;
+    color: #ccc;
+    font-size: 20px;
+    word-wrap:break-word;
+    margin: 1em 10% 0 10%;
```
Comment (roboshoes) [inline app/assets/stylesheets/views/tracks_screen_view.css.scss]: grundsätzlich nicht falsch, aber ich finde `em` und `%`sollten wir nicht wirklich benutzen wenn wir alles andere auch pixel basiert machen. Muss jetzt nich ausgebessert werden. Aber ich werfs trotzdem mal so rein.

**Comment by roboshoes** [comment]
Kleine syntax sachen. sonst ists nett!

File: app/assets/javascripts/slotcars/tracks/templates/page_view_template.js.hjs (hunk 1 above)
Comment (DominikGuzei) [inline app/assets/javascripts/slotcars/tracks/templates/page_view_template.js.hjs]: you got me! :-D

Comment (DominikGuzei) [inline app/assets/javascripts/slotcars/tracks/templates/page_view_template.js.hjs]: of course this was just a test, if you bastards look at it at all! :wink:

File: app/assets/stylesheets/views/tracks_screen_view.css.scss (hunk 2 above)
Comment (DominikGuzei) [inline app/assets/stylesheets/views/tracks_screen_view.css.scss]: I removed the em / % and replaced them with pixels!

**Pull request merged**